modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -94,6 +95,22 @@ enum class GShape: int
     GFRAME,
 };
 
+namespace gapi {
+namespace s11n {
+namespace detail {
+template<typename T> struct wrap_serialize
+{
+    static void serialize(gapi::s11n::IOStream& os, const util::any& arg)
```

Comment:
Yes, it could, but this way https://github.com/opencv/opencv/pull/18496/files#diff-a92ca3714491807461fbd9ae39362adeR332
operator>>(IOStream os, GCompileArg carg) will do carg.serialize(os, carg)